iothub/device/src/WritablePropertyResponse.cs

```diff
@@ -53,5 +67,11 @@ public WritablePropertyResponse(object propertyValue, int ackCode, long ackVersi
         /// </summary>
         [JsonProperty("ad", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string AckDescription { get; set; }
+
+        /// <summary>
+        /// The serialized property value.
+        /// </summary>
+        [JsonProperty("value")]
+        public JRaw ValueAsJson => new JRaw(_objectSerializer.SerializeToString(Value));
```

Comment:
Does this tie the writable property too closely with Newtonsoft? Seeing as how we want to decouple this the serialization shouldn't happen here but when the byte stream for the resulting message is constructed.

Reply:
writable property response could have serialization happening as per two different settings:
- one for the user-supplied value, which could be any object and thus could bring in its own serialization options.
- second for the serialization of the "writablepropertyresponse" type, which does have newtonsoft annotations for its properties (we own it). 

The JRaw ValueAsJson serializes the value as per the serializer provided by the user and stores it as "raw" string. This value isn't serialized again by newtonsoft when serializing the "writablepropertyresponse".